### ISSUE TITLE
Defining a `char` gives a Norm error

### ISSUE THREAD

**Issue by herbievine** [42School/norminette#407]
### Describe the bug
Defining a `char` gives a Norm error since v3.3.53.

```c typedefs.h
# define A 'A'
// Error: Preprocessor statement must only contain constant defines
```

The issue in my CI/CD:
https://github.com/herbievine/42/actions/runs/5532311412/jobs/10094211076?pr=33

**Comment by NiumXp** [comment]
The same happens in a `#if`.

**Comment by herbievine** [comment]
Wow, that was quick. What's the ETA on this being released? This is causing havoc on all my builds lol...

**Comment by NiumXp** [comment]
We need to wait for someone to accept the fix and then upload a new version to PyPi. Sorry, I completely forgot to deal with characters when updating the preprocessors stuff.

Temporary suggestions to "fix" this problem:
- You can use norminette from my branch [fix/#408](https://github.com/NiumXp/norminette/tree/fix/%23407)
- You can replace characters with ints: `#define A 65`
